controller.go

```diff
@@ -357,6 +357,10 @@ func (c *controller) NewNetwork(networkType, name string, options ...NetworkOpti
 		return nil, err
 	}
 
+	if network.Type() == "bridge" && network.isolated {
+		return nil, ErrNetworkNotSupportIsolation(network.Type())
```

Comment:
@chenchun 
Yes, let's think a bit more about the design.
- There is a requirement for user to be able to specify that the network must be isolated (this is taken care by your `NetworkOptionIsolatedNetwork` + changes in bridge driver)
- Then there is a requirement for not adding the default gw network along to remote driver network

Covering the second requirement via isolated network may not be the cleanest way.
I am thinking the driver should let libnetwork know at registration time.